docs/en/latest/FAQ.md

```diff
@@ -502,3 +502,37 @@ $ acme.sh --renew --domain demo.domain
 ```
 
 Blog https://juejin.cn/post/6965778290619449351 has detail setup.
+
+## How to remove route prefix for path matching
```

Comment:
Also, need to update the Chinese version.

Reply:
Thanks for reminding! Done